api/estados/tocantins.js

```diff
@@ -1,146 +1,148 @@
-  "Porto Nacional",
-  "Praia Norte",
-  "Presidente Kennedy",
-  "Pugmil",
-  "Recursolândia",
-  "Riachinho",
-  "Rio da Conceição",
-  "Rio dos Bois",
-  "Rio Sono",
-  "Sampaio",
-  "Sandolândia",
-  "Santa Fé do Araguaia",
-  "Santa Maria do Tocantins",
-  "Santa Rita do Tocantins",
-  "Santa Rosa do Tocantins",
-  "Santa Tereza do Tocantins",
-  "Santa Terezinha Tocantins",
-  "São Bento do Tocantins",
-  "São Félix do Tocantins",
-  "São Miguel do Tocantins",
-  "São Salvador do Tocantins",
-  "São Sebastião do Tocantins",
-  "São Valério da Natividade",
-  "Silvanópolis",
-  "Sítio Novo do Tocantins",
-  "Sucupira",
-  "Taguatinga",
-  "Taipas do Tocantins",
-  "Talismã",
-  "Tocantínia",
-  "Tocantinópolis",
-  "Tupirama",
-  "Tupiratins",
-  "Wanderlândia",
-  "Xambioá"
-  ]
+const to = {
+  state: 'Tocantins',
+  abbreviation: 'to',
+  cities: [
+    'Abreulândia',
+    'Aguiarnópolis',
+    'Aliança do Tocantins',
+    'Almas',
+    'Alvorada',
+    'Ananás',
+    'Angico',
+    'Aparecida do Rio Negro',
+    'Aragominas',
+    'Araguacema',
+    'Araguaçu',
+    'Araguaína',
+    'Araguanã',
+    'Araguatins',
+    'Arapoema',
+    'Arraias',
+    'Augustinópolis',
+    'Aurora do Tocantins',
+    'Axixá do Tocantins',
+    'Babaçulândia',
+    'Bandeirantes do Tocantins',
+    'Barra do Ouro',
+    'Barrolândia',
+    'Bernardo Sayão',
+    'Bom Jesus do Tocantins',
+    'Brasilândia do Tocantins',
+    'Brejinho de Nazaré',
+    'Buriti do Tocantins',
+    'Cachoeirinha',
+    'Campos Lindos',
+    'Cariri do Tocantins',
+    'Carmolândia',
+    'Carrasco Bonito',
+    'Caseara',
+    'Centenário',
+    'Chapada da Natividade',
+    'Chapada de Areia',
+    'Colinas do Tocantins',
+    'Colméia',
+    'Combinado',
+    'Conceição do Tocantins',
+    'Couto de Magalhães',
+    'Cristalândia',
+    'Crixás do Tocantins',
+    'Darcinópolis',
+    'Dianópolis',
+    'Divinópolis do Tocantins',
+    'Dois Irmãos do Tocantins',
+    'Dueré',
+    'Esperantina',
+    'Fátima',
+    'Figueirópolis',
+    'Filadélfia',
+    'Formoso do Araguaia',
+    'Fortaleza do Tabocão',
+    'Goianorte',
+    'Goiatins',
+    'Guaraí',
+    'Gurupi',
+    'Ipueiras',
+    'Itacajá',
+    'Itaguatins',
+    'Itapiratins',
+    'Itaporã do Tocantins',
+    'Jaú do Tocantins',
+    'Juarina',
+    'Lagoa da Confusão',
+    'Lagoa do Tocantins',
+    'Lajeado',
+    'Lavandeira',
+    'Lizarda',
+    'Luzinópolis',
+    'Marianópolis do Tocantins',
+    'Mateiros',
+    'Maurilândia do Tocantins',
+    'Miracema do Tocantins',
+    'Miranorte',
+    'Monte do Carmo',
+    'Monte Santo do Tocantins',
+    'Muricilândia',
+    'Natividade',
+    'Nazaré',
+    'Nova Olinda',
+    'Nova Rosalândia',
+    'Novo Acordo',
+    'Novo Alegre',
+    'Novo Jardim',
+    'Oliveira de Fátima',
+    'Palmas',
+    'Palmeirante',
+    'Palmeiras do Tocantins',
+    'Palmeirópolis',
+    'Paraíso do Tocantins',
+    'Paranã',
+    "Pau d'Arco",
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)